### ISSUE TITLE
Focus with keyboard to message bar Hard to see

### ISSUE THREAD

**Issue by chrisdias** [Microsoft/vscode#3769]
At least, I can't find a way to set focus there so that I can navigate to the various options and choose one.

**Comment by bpasero** [comment]
@chrisdias it is possible but the focus indication for buttons is not helpful since the buttons are already blue. 

@bgashler1 maybe you could take a look into better focus indication here? we seem to have 2 classes for buttons, the one being `action-button` (in the message box) and `monaco-button` (when you open an empty workspace and open the explorer where we have a button for "Open Folder").

I group focus rules in shell.css (https://github.com/Microsoft/vscode/blob/master/src/vs/workbench/electron-browser/media/shell.css#L50)

I am also open to suggestions for changing the outline for actions (currently they also show a subtle blue outline border).

**Comment by bgashler1** [comment]
@bpasero I'm not able to get focus onto these buttons, even in the "Tab moves focus" mode. Perhaps we need to create an issue on this?

However, I did add styling rules for them.  I manually pinned the focus using Developer Tools.

<img width="517" alt="screen shot 2016-03-16 at 4 59 38 pm" src="https://cloud.githubusercontent.com/assets/11839736/13832397/20c17406-eb99-11e5-84c2-1b1cd2cc52fb.png">
<img width="512" alt="screen shot 2016-03-16 at 5 01 41 pm" src="https://cloud.githubusercontent.com/assets/11839736/13832399/20e4336a-eb99-11e5-87b2-28d15926d603.png">
<img width="546" alt="screen shot 2016-03-16 at 5 02 36 pm" src="https://cloud.githubusercontent.com/assets/11839736/13832398/20e34252-eb99-11e5-92be-a79b35ee2068.png">

**Comment by chrisdias** [comment]
assigning to @bpasero for the focus issue.

**Comment by bgashler1** [comment]
The style change is in PR #4339

**Comment by bpasero** [comment]
@bgashler1 that is weird, I can press the "Tab" key and eventually reach the message bar.

I like the color for the button in dark theme, however in light theme I cannot see the focus indication. I suggest we come up with a new color here, maybe towards orange even?

**Comment by bpasero** [comment]
(try it out on the "Open Folder" button in the explorer when you have no folder opened)

**Comment by chrisdias** [comment]
i was able to get focus to the message bar on Windows (didn't try OSX yet) but i can't move between the buttons. Tab moves me off the message bar, left and right arrow don't do anything.

![image](https://cloud.githubusercontent.com/assets/1487073/13865009/e91e06b0-ec63-11e5-8105-d3ca9b7f419d.png)

opened 4396 on this.

**Comment by egamma** [comment]
assigning to Joao to verify.
